testing/run-scripts/serve-zork-metrics.sh

```diff
@@ -0,0 +1,4 @@
+docker build -t zork-metrics ../../monitor
+HOST_IP=`ip -o -4 addr list docker0 | awk '{print $4}' | cut -d/ -f1`
+docker run -d -p 8000:8000 --add-host zork:$HOST_IP zork-metrics
```

Comment:
this is giving the error:
`Conflicting options: host type networking can't be used with links. This would result in undefined behavior.`
I'm still digging!